docs/sources/fundamentals/getting-started/first-dashboards/get-started-grafana-ms-sql-server.md

```diff
@@ -1,8 +1,9 @@
 ---
 aliases:
-  - ../guides/getting_started/
-  - ../guides/gettingstarted/
-  - getting-started-sql/
+  - ../../../guides/getting_started/ # /docs/grafana/latest/guides/getting_started
+  - ../../../guides/gettingstarted/ # /docs/grafana/latest/guides/gettingstarted
```

Comment:
These are duplicates of the aliases in `docs/sources/fundamentals/getting-started/first-dashboards/_index.md`
```suggestion
```